src/main/java/com/virtudoc/web/controller/Notification.java

```diff
@@ -0,0 +1,63 @@
+package com.virtudoc.web.controller;
+import java.time.LocalDateTime;
+
+public class Notification {
+    private String patientName;
+    private String doctorName;
+    private String location;
+    private String reasonForVisit;
+//    todo: convert to Date object
+    private String date;
```

Comment:
Did you want to convert this to a date object, or are you good keeping it a string?

Reply:
I just used a string for the first sprint to have something there. I can talk with Will once he has the "Create Appointment" form up since both should use the same Date formatting.